kn/kn-create.yaml

```diff
@@ -18,11 +18,9 @@ spec:
     image: gcr.io/knative-releases/github.com/knative/client/cmd/kn
     command: ["/ko-app/kn"]
     args:
-    - service
-    - create
-    - ${inputs.params.service}
-    - --image
-    - ${inputs.resources.image.url}
-    - --force
```

Comment:
The only space separated flag kn didnt like was this bool flag `--force`. This worked for me
```
✗  git --no-pager diff                           
diff --git a/kn/kn-create.yaml b/kn/kn-create.yaml
index 209fc47..69e79ab 100644
--- a/kn/kn-create.yaml
+++ b/kn/kn-create.yaml
@@ -23,6 +23,5 @@ spec:
     - ${inputs.params.service}
     - --image
     - ${inputs.resources.image.url}
-    - --force
-    - ${inputs.params.force}
+    - --force=${inputs.params.force}
     # TODO(jasonhall): Support limits/resources, env vars, etc.
```

Reply:
Ah interesting, good to know. I think I'd prefer to keep it like this now, so they're consistently using `=` where possible.

The error message is not terribly helpful in this case, would you like me to file a bug for knative/client?

Reply:
> The error message is not terribly helpful in this case, would you like me to file a bug for knative/client ?

That'd be helpful, thanks! :-)

Reply:
@ImJasonH : On the same note, IMO we don't need single quotes. WDYT ?